docker-images/syntax-highlighter/Cargo.toml

```
@@ -1,16 +1,18 @@
 [package]
 name = "syntect_server"
-version = "1.0.0"
+version = "1.0.1"
 authors = ["Stephen Gutekanst <stephen.gutekanst@gmail.com>"]
 edition = "2018"
 license = "MIT"
 
 [dependencies]
-syntect = { git = "https://github.com/slimsag/syntect" }
-rocket = { git = "https://github.com/SergioBenitez/Rocket" } # Rocket 0.5 is not released yet: https://github.com/SergioBenitez/Rocket/issues/19#issuecomment-684122645
-rocket_contrib = { git = "https://github.com/SergioBenitez/Rocket" } # Rocket 0.5 is not released yet: https://github.com/SergioBenitez/Rocket/issues/19#issuecomment-684122645
+syntect = { git = "https://github.com/sourcegraph/syntect" }
```

Something like this is probably more idiomatic, although I don't have strong feelings:

```suggestion
syntect = { git = "https://github.com/sourcegraph/syntect" }
rocket = { git = "https://github.com/SergioBenitez/Rocket", features = ["json"] } # Rocket 0.5 is not released yet: https://github.com/SergioBenitez/Rocket/issues/19#issuecomment-684122645
```

---

Added `fixup!` commit with this tweak.